ask.py

```diff
@@ -735,10 +736,10 @@ def startup(file_name):
 			app.db.create_all()
 			print('\t✅')
 
-		# Stats the local development server.
+		# Starts the local development server.
 		# TODO: ALso support running the app in a production ready server.
 		style_print('Running Flask app:', styles=['bold'])
-		os.system('export FLASK_APP=app.py')
+                os.environ['FLASK_APP']='app.py'
```

Comment:
Remove the indent before `os.`. Put spaces around the equals sign.

Reply:
apologies for that. let me do that.

Reply:
done. everything should be fine now